src/components/ShareStructure/ShareStructures.vue

```diff
@@ -49,23 +51,27 @@ import {
 export default class ShareStructures extends Mixins(CommonMixin) {
   /** Edit Mode */
   @Prop({ default: true })
-  private isEditMode!: string
+  readonly isEditMode!: string
 
-  // Global Getters
+  // Global getters
   @Getter getNewResolutionDates!: string []
   @Getter getOriginalIA!: IncorporationFilingIF
   @Getter getShareClasses!: ShareClassIF[]
-  @Getter getOriginalSnapshot!: BusinessSnapshotIF[]
+  @Getter getOriginalSnapshot!: BusinessSnapshotIF
 
-  // Global Actions
+  // Global actions
   @Action setShareClasses!: ActionBindingIF
   @Action setShareStructureChanged!: ActionBindingIF
   @Action setEditingShareStructure!: ActionBindingIF
 
-  // Local Properties
-  private toggleResolutionDateDialog = false
+  // Local property
+  toggleResolutionDateDialog = false
+
+  get originalShareStructure (): ShareStructureIF {
+    return this.getOriginalSnapshot?.shareStructure
```

Comment:
This was needed because original snapshot can be null.

Reply:
looks good